docs/advanced/classes.rst

```diff
@@ -490,6 +492,26 @@ you could equivalently write:
 
 which will invoke the constructor in-place at the pre-allocated memory.
 
+Brace initialization
+--------------------
+
+``pybind11::init<>`` internally uses C++11 brace initialization to call the
+constructor of the target class. This means that it can be used to bind
+*implicit* constructors as well:
+
+.. code-block:: cpp
+
+    struct POD {
+        int a;
+        std::string b;
+    };
+
+    py::class_<POD>(m, "POD")
+        .def(py::init<int, const std::string &>());
+
+Note that brace initialization preferentially invokes constructor overloads
+taking a ``std::initializer_list``.
```

Comment:
Perhaps add:

In the rare event that this causes an issue, you can work around it by using `py::init(...)` with a lambda function that constructs the new object as desired.

Since that makes it more than just a single sentence, it's probably useful to put it in a `note` section.